<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>376</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>376</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: white
</string>
  </property>
  <widget class="QWidget" name="sign_in_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>376</width>
     <height>800</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>376</width>
     <height>800</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>376</width>
     <height>800</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#bgwidget {
background-color: #ffffff;
}</string>
   </property>
   <widget class="QPushButton" name="contin">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>640</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 12pt &quot;MS Shell Dlg 2&quot;;
	color: 'white';
	border-radius: 10px;
	background-color: rgb(5, 140, 249)
}

QPushButton:hover {
	background-color: rgb(43, 157, 251)
}</string>
    </property>
    <property name="text">
     <string>Continue</string>
    </property>
   </widget>
   <widget class="QLabel" name="footer">
    <property name="geometry">
     <rect>
      <x>-3</x>
      <y>760</y>
      <width>384</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
color: '#ffffff';
background-color: rgb(39, 35, 103)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Company Name, Location&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernamefield">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color:rgba(0,0,0,0);
left: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordfield">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>570</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgba(0,0,0,0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>550</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Sign In</string>
    </property>
   </widget>
   <widget class="QLabel" name="processinput">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>616</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: 'red';
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="logo_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>331</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>icons/gui/logo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: '#ffffff';
background-color: rgb(39, 35, 103)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Product Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="go_to_register">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>710</y>
      <width>142</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	color: 'black';
	border-radius: 10px;
	background-color: 'white'
}

</string>
    </property>
    <property name="text">
     <string>Create an account</string>
    </property>
   </widget>
   <zorder>go_to_register</zorder>
   <zorder>contin</zorder>
   <zorder>footer</zorder>
   <zorder>usernamefield</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>processinput</zorder>
   <zorder>passwordfield</zorder>
   <zorder>logo_2</zorder>
   <zorder>label_11</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>usernamefield</tabstop>
  <tabstop>passwordfield</tabstop>
  <tabstop>contin</tabstop>
  <tabstop>go_to_register</tabstop>
 </tabstops>
 <resources>
  <include location="logos.qrc"/>
 </resources>
 <connections/>
</ui>
